servers/visual_server.h

```diff
@@ -1093,9 +1093,15 @@ class VisualServer : public Object {
 
 	/* EVENT QUEUING */
 
+	enum ChangedPriority {
+		CHANGED_PRIORITY_HIGH = 0,
+		CHANGED_PRIORITY_LOW,
+		CHANGED_PRIORITY_ANY,
+	};
```

Comment:
Since `ANY` is the default behavior, maybe it should take id 0?

Reply:
Yes can do. :+1: